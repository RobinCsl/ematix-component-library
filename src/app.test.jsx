import React from "react";
import renderer from "react-test-renderer";

import App from "./app";

it("renders correctly", () => {
  const tree = renderer
    .create(<div><App /></div>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
