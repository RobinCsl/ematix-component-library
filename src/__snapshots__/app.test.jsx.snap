// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className={undefined}
  >
    <div
      className={undefined}
    >
      <img
        alt="logo"
        className={undefined}
        src="logo.svg"
      />
      <div
        className={undefined}
      >
        component-library
      </div>
      <div>
        <div>
          <button>
            click me!
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
